### PR TITLE
More precise language and example

### PR DESCRIPTION
**Pull request by AndreasWolter**

### CODE REVIEW

**Comment by AndreasWolter** [comment]
[microsoft-github-policy-service](https://github.com/apps/microsoft-github-policy-service) What do you want from me?

**Comment by Court72** [comment]
@AndreasWolter,

Thank you for your contribution. Would you take a moment to sign the [Contributor License Agreement (CLA)](https://github.com/MicrosoftDocs/sql-docs/pull/9944#issuecomment-2495061612)? After the CLA is signed, someone can review your pull request. Thanks!

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by AndreasWolter** [comment]
@Court72 thanks. I just don't know what I should chose here. This legal stuff is confusing to me. I have contributed to numerous doc articles before without this question. I do not work for Microsoft, but for Sarpedon Quality Lab LLC (Which I own). But what this regarding to rights to content here is above my understanding..
Could you please dumb it down for me? :)

**Comment by rwestMSFT** [comment]
@AndreasWolter I think this is pretty clear:

Screenshot 2024-11-25 at 09 39 10

**Comment by AndreasWolter** [comment]
if you say so @rwestMSFT . For me this is work. I mean it's certainly not private time. And give that I am the owner of my company its hard to distinguish whether I own it as myself or as my company. 

therefore this seems to fit equally:
(when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.

So it comes down to: what is the intended effect of this? What difference does it make? I mean there must be a difference after all?

**Comment by AndreasWolter** [comment]
anyway. I don't want to waste our time texting about this. So let me just do it this way and see what happens, if anything and be done with it :)
@microsoft-github-policy-service agree company="Sarpedon Quality Lab LLC"

**Comment by rwestMSFT** [comment]
Finally given me a reason to refresh two other audit articles as well, thank you :)

**Comment by AndreasWolter** [comment]
happy to help

**Comment by rwestMSFT** [comment]
> So it comes down to: what is the intended effect of this? What difference does it make? I mean there must be a difference after all?

It comes down to legal liability. Because we take external contributions for our content, it needs proper attribution.

**Comment by AndreasWolter** [comment]
right. still not sure what the actual effect would be. I guess in worst case MS talks to my company's representative. And from my end the LLC protects me :)

**Comment by rwestMSFT** [comment]
No effect as such. It's just a legal requirement for US copyright law.

**Comment by AndreasWolter** [comment]
Ha. So my feeling wasn't totally wrong :)
